Description: Don't copy test suite output to a log.
 When running the test suite, don't copy stdout and stderr output to a log
 file, as this would break under LC_ALL=C, PYTHONWARNINGS=d and python3.X.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-09-18

--- a/test/alltests.py
+++ b/test/alltests.py
@@ -69,9 +69,6 @@
     sys.stdout.flush()
     return suite
 
-# must redirect stderr *before* first import of unittest
-sys.stdout = sys.stderr = Tee('alltests.out')
-
 import package_unittest
 
 
